### PR TITLE
Add institution field to APIs returning participants 

### LINKED ISSUE #311: Add institution field to APIs returning participants
The field should contain the readable name after the join to the Institution table and not the primary key. See how the Analysis-User relationship and the pipeline supported type relationships are used.

### PR DESCRIPTION
**Pull request by Madelinehazel**
closes #311
 Not sure what 'See how the Analysis-User relationship and the pipeline supported type relationships are used.' in the issue description is referring to.

### CODE REVIEW

**Comment by kevinlul** [comment]
I was referring to how a custom property could be used to retrieve the desired value across a relationship

https://github.com/ccmbioinfo/stager/blob/master/flask/app/models.py#L378-L382

File: flask/app/participants.py [hunk 1]
```diff
@@ -237,6 +243,7 @@ def create_participant():
         affected=request.json.get("affected"),
         solved=request.json.get("solved"),
         participant_type=request.json.get("participant_type"),
+        institution_id=request.json.get("institution_id"),
```
Comment (kevinlul) [inline flask/app/participants.py:246]: Can you make this endpoint accept the text form of institution instead and do the same for `/api/_bulk`?

Comment (Madelinehazel) [reply flask/app/participants.py]: revised in 4f944d550680490e9aded34026c3e4d37721f918

File: flask/app/participants.py [hunk 2]
```diff
@@ -235,6 +235,20 @@ def create_participant():
...
+    if institution:
+        institution_obj = models.Institution.query.filter(
+            models.Institution.institution == institution
+        ).one_or_none()
+        if institution_obj:
+            institution_id = institution_obj.institution_id
+        else:
+            institution_obj = models.Institution(institution=institution)
+            db.session.add(institution_obj)
+            transaction_or_abort(db.session.commit)
```
Comment (kevinlul) [inline flask/app/participants.py:249]:
```suggestion
            transaction_or_abort(db.session.flush)
```

**Review by kevinlul — APPROVED** [review]
LGTM

**Pull request merged**